#!/bin/sh /usr/share/dpatch/dpatch-run
## 201-magic-update-mp3.dpatch by Werner Fink <werner@suse.de>
##
## DP: Fixes regressions with mp3 files (Closes: #480683, #481377).

@DPATCH@

diff -Naurp file-4.24.orig/magic/Magdir/animation file-4.24/magic/Magdir/animation
--- file-4.24.orig/magic/Magdir/animation	2008-05-04 14:12:49.000000000 +0000
+++ file-4.24/magic/Magdir/animation	2008-05-21 09:23:29.000000000 +0000
@@ -276,7 +276,7 @@
 # modified by Joerg Jenderek
 # GRR the original test are too common for many DOS files
 # so don't accept as MP3 until we've tested the rate
-0       beshort&0xFFFE
+0	beshort&0xFFFE 0xFFFA
 # rates
 >2      byte&0xF0       0x10           MPEG ADTS, layer III, v1,  32 kbps
 !:mime	audio/mpeg
